src/Shared/Components/CICDHistory/DeploymentStatusBreakdown.tsx

```diff
@@ -0,0 +1,94 @@
+import { useRouteMatch } from 'react-router'
+import { TIMELINE_STATUS, URLS } from '../../../Common'
+import { ErrorInfoStatusBar } from './ErrorInfoStatusBar'
+import { DeploymentStatusDetailRow } from './DeploymentStatusDetailRow'
+import { DeploymentStatusDetailBreakdownType } from './types'
+import { getAppDetails } from './utils'
+import ErrorBar from '../Error/ErrorBar'
+
+const DeploymentStatusDetailBreakdown = ({
+    deploymentStatusDetailsBreakdownData,
+    isVirtualEnvironment,
+}: DeploymentStatusDetailBreakdownType) => {
+    const _appDetails = getAppDetails()
```

Comment:
References to wrong index store

Reply:
migrated IndexStore from dashboard, please review that also